src/clue.js

```diff
@@ -1,94 +1,163 @@
+age:          62,
+image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+occupation:   "Retired Football player"
+}
+// Weapons
+var rope = {
+name: "rope",
+weight: 10
+}
+var knife = {
+name: "knife",
+weight: 8
+}
+var candlestick = {
+name: "candlestick",
+weight: 2
+}
+var dumbbell = {
+name: "dumbbell",
+weight: 30
+}
+var poison = {
+name: "poison",
+weight: 2
+}
+var axe = {
+name: "axe",
+weight: 15
+}
+var bat = {
+name: "bat",
+weight: 13
+}
+var trophy = {
+name: "trophy",
+weight: 25
+}
+var pistol = {
+name: "pistol",
+weight: 20
+}
+// Rooms
+var dinningroom = {
+name: "Dinning Room"}
+var conservatory = {
+name: "Conservatory"}
+var kitchen = {
+name: "Kitchen"}
+var study = {
+name: "Study"}
+var library = {
+name: "Library"}
+var billiardroom = {
+name: "Billiard Room"}
+var lounge = {
+name: "Lounge"}
+var ballroom = {
+name: "Ballroom"}
+var hall = {
+name: "Hall"}
+var spa = {
+name: "Spa"}
+var livingroom = {
+name: "Living Room"}
+var observatory = {
+name: "Observatory"}
+var theater = {
+name: "Theater"}
+var guesthouse = {
+name: "Guest House"}
+var patio = {
+name: "Patio"}
 
-missScarlet
-first_name:   Kasandra
-last_name:    Scarlet
-color:        red
-description:  She is an A-list movie star with a dark past
-age:          31
-image:        https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg
-occupation:   Actor
+// Characters Collection
+var charactersArray = [mrGreen,drOrchid,profPlum,missScarlet,mrsPeacock,mrMustard];
 
-mrsPeacock
-first_name:   Eleanor
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
+// Rooms' Collection
+var roomsArray = [dinningroom,conservatory,kitchen,study,library,billiardroom,lounge,ballroom,hall,spa,livingroom,observatory,theater,guesthouse,patio];
 
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+// Weapons Collection
+var weaponsArray = [rope,knife,candlestick,dumbbell,poison,axe,bat,trophy,pistol];
 
-// Weapons
+function randomSelector(card){
+  var randomselect = Math.floor(Math.random() * card.length);
+  return card[randomselect]
+}
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+function pickMistery(){
+  var char = randomSelector(charactersArray);
```

Comment:
What is ```char```? I would name it characterChoosen or something like that. It can be confusing if not